fastlane/Fastfile

```diff
@@ -43,6 +43,8 @@ SUPPORTED_LOCALES = [
   { glotpress: "zh-tw", google_play: "zh-TW",  promo_config: {} },
 ].freeze
 
+UPLOAD_TO_PLAY_STORE_JSON_KEY = File.join(Dir.home, '.configure', 'simplenote-android', 'secrets', 'google-upload-credentials.json')
```

Comment:
I didn't really need to extract the path in this constant, because it's used only once. Still, it keeps the code below clearer and it will make it easier to reuse in the future.